examples/cpp/content_filter/CustomContentFilter.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef _FASTDDS_CONTENT_FILTER_CUSTOM_FILTER_HPP_
-#define _FASTDDS_CONTENT_FILTER_CUSTOM_FILTER_HPP_
+#ifndef FASTDDS_EXAMPLES_CPP_CONTENT_FILTER__CUSTOM_CONTENT_FILTER_HPP
```

Comment:
Pls apply in the rest of the file
```suggestion
#ifndef FASTDDS_EXAMPLES_CPP_CONTENT_FILTER__CUSTOMCONTENTFILTER_HPP
```